src/main.rs

```diff
@@ -57,20 +57,41 @@ async fn main() {
 
     // Get config from config path
     // Options set in environment variables override the properties from config file
-    let config = Config::builder()
-        .add_source(config::File::with_name(&config_path))
-        .add_source(config::Environment::default())
+
+    // Use ';' as separator, as argon hashes contain commas
+    let env_source = config::Environment::default()
+        .list_separator(";")
+        .with_list_parse_key("API_KEY_HASHES")
+        .with_list_parse_key("CORS_ALLOWED_ORIGINS")
+        .with_list_parse_key("CORS_ALLOWED_METHODS")
+        .try_parsing(true);
+
+    let mut config_builder = Config::builder();
+    // Make environment only configuration possible
+    if Path::new(&config_path).exists() {
+        config_builder = config_builder.add_source(config::File::with_name(&config_path));
+    }
```

Comment:
Checked das nicht die lib schon? Was war davor verpflichtend in der config Datei?

Reply:
Das checked die lib schon, nur leider paniced sie wenn die Datei nicht gefunden werden kann

Reply:
Unschön, aber ergibt auch irgendwo Sinn. Will man hier in dem Fall evtl. einfach `required: false` setzen? Denke dann kann man sich den check sparen

Reply:
ah hatte das nicht gefunden, sounds good